tests/memarray/main.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2017 Inria
```

Comment:
2018 Tobias Heider <heidert@nm.ifi.lmu.de> ;-)

Reply:
That one was initially done by @rfuentess so I don't want to change it without his consent. 
I can update the date though.

Reply:
For me is OK to share authorship or given it to @tobhe, after all he had to upgrade the code.